src/application-navigation/ApplicationNavigation.jsx

```diff
@@ -44,6 +44,10 @@ const propTypes = {
    * An element to render within the ApplicationNavigation utility menu, shifted to the drawer at the `medium` breakpoint and below.
    */
   hero: PropTypes.element,
+  /**
+   * The base id used to generate ids of workspace, navigation, utility, and extension items
+   */
+  id: PropTypes.string,
```

Comment:
I know we cannot make this required passively, but it _is_ required. Are we generating an id somewhere if this isn't provided?

Reply:
Updated.